src/lib/validation/utils/helpers.js

```diff
@@ -16,6 +16,7 @@ function shouldValidate (input) {
     isRequired ||
     input.hasAttribute('pattern') ||
     input.getAttribute('type') === 'email' ||
+    input.hasAttribute('data-ad-id') ||
```

Comment:
where comes this from? Why is this needed?

Reply:
the Datepicker uses this specific [attribute](https://github.com/mathislucka/a11y-datepicker/blob/master/src/js/draw/Draw.js#L27). See here a comment from @spiess-demos https://github.com/demos-europe/demosplan-ui/pull/841#discussion_r1587279898 . I think it is a good idea to get the Datepicker input with this attribute, but I think I should add a comment for a better readability

Reply:
Ok. Thanks. The Comment helps a lot